src/sass/patterns/_form-elements.scss

```diff
@@ -1,16 +1,52 @@
 // Form Elements
 // =============
 
-.vertical-separator {
-  border-right: 1px solid $color-border-separator-alt;
+.form-grid {
+  display: grid;
+  // needs a non-percentage and non fr
+  //   for changes column to be consistent between row types
+  //   16vw is a magic number that works well in changes column
+  grid-template-columns: 1fr 16vw;
+}
```

Comment:
If this is specific to the task-capture / changes layout, should it be nested under `[data-form='task-capture']`?

Reply:
only if this class is used other places where we don't want the behavior. But I think this isa new class created specifically for this. Could change the name to `task-capture-form` or  something if it needs more clarity, rather than nesting.